freqtrade/worker.py

```diff
@@ -90,6 +90,9 @@ def _worker(self, old_state: Optional[State]) -> State:
             if state == State.RUNNING:
                 self.freqtrade.startup()
 
+            if state == State.STOPPED:
+                self.freqtrade.check_for_open_trades()
+
```

Comment:
I would move this line 4(ish) lines below - where the same `if` statement is used already...

Reply:
Also said it on Slack, but for other people reading this: putting the if-statement here makes it only run once (when the state transitions), if you place it below, it would spam you on the Telegram output as long as the bot's state is `stopped`.